### DISCUSSION TITLE
Accessing login page invalidates session

### DISCUSSION THREAD

**Discussion by TanguyHefnaoui** [codegreencreative/laravel-samlidp#132] [Q&A]
I am using this library to authenticate users before they access our product's doc360 documentation (behind sso auth).
I want to make it so if our users are already logged in (which will be in most cases) the sso authentication happens automatically. However I am running into some issues, when accessing the login page from the service provider auth request, the users are always logged out (session seems to be invalidated when accessing the page i.e. auth()->user() === null). 
Any idea what could be causing this?

I am able to go through the sso process just fine but I am always logged out of the app when the SP sends me to my login page

I am running laravel 10 with a vue front end

**Comment by upwebdesign** [comment]
@TanguyHefnaoui, this package creates an IdP. It is not meant for service providers. If you are using this package as an IdP, and are requesting to log into an SP, it is likely a configuration issue. I would guess from the SP side of things.

**Comment by TanguyHefnaoui** [comment]
Having my sp send the request via get instead of post solved my issue

**Comment by upwebdesign** [comment]
@TanguyHefnaoui, fantastic, I'm happy you figured it out!
